plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/IcebergOptimizeHandle.java

```diff
@@ -108,6 +111,12 @@ public boolean isRetriesEnabled()
         return retriesEnabled;
     }
 
+    @JsonProperty
+    public boolean isWholeTableScan()
```

Comment:
Maybe we can do https://github.com/trinodb/trino/pull/12964 to avoid adding this field here.

Reply:
I really like the idea!